src/notyf.ts

```diff
@@ -48,7 +48,9 @@ export default class Notyf {
     const duration = notification.options.duration || this.options.duration;
     setTimeout(() => {
       const index = this.notifications.indexOf(notification);
-      this.notifications.splice(index, 1);
+      if(index!=1){
```

Comment:
I think you meant to evaluate against `-1`, right?
```suggestion
      if(index !== -1){
```